tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "moduleResolution": "node",
     "allowSyntheticDefaultImports": true,
     "allowJs": true,
-    "importHelpers": true,
+    "importHelpers": false,
```

Comment:
Why is this change needed?

Reply:
we were talking to avoid installing tslib npm package, actually importHelpers use this package to reduce the size of output, https://webhint.io/docs/user-guide/hints/hint-typescript-config/import-helpers/

By default in typescript this option is set to false.

Reply:
Can you also confirm this is also false in Recco project as well?

Reply:
ok

Reply:
yes @jjalan  sir, I have confirmed it we are not setting this flag ```importHelpers``` in recco project